tests/messages/test_extract.py

```diff
@@ -60,6 +60,15 @@ def test_nested_comments(self):
                                                ['TRANSLATORS:'], {}))
         self.assertEqual([(1, 'ngettext', (u'pylon', u'pylons', None), [])],
                          messages)
+        buf = BytesIO(b"""\
```

Comment:
These tests are nice enough, but can you move them to another function, `def test_npgettext(self):` for instance? :)